packages/fxa-settings/src/components/Settings/FlowRecoveryKeyHint/index.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react';
+import { FlowContainer } from '../FlowContainer';
+import { ProgressBar } from '../ProgressBar';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { useForm } from 'react-hook-form';
+import InputText from '../../InputText';
+import { useFtlMsgResolver, useAccount } from '../../../models';
+import { logViewEvent, usePageViewEvent } from '../../../lib/metrics';
+import { LightbulbImage } from '../../images';
+
+export type FlowRecoveryKeyHintProps = {
+  navigateForward: () => void;
+  navigateBackward: () => void;
+  localizedPageTitle: string;
+};
+
+type HintForm = { hint: string };
+
+export const viewName = 'settings.account-recovery.hint';
+export const saveHintSuccessEventName = 'save-hint-success';
+export const saveHintFailureEventName = 'save-hint-failure';
+export const backwardNavigationEventName = 'navigate-backward';
```

Comment:
For review in the metrics ticket - I'm not sure if we need/want to log the navigate back/forward. Maybe more informative events like abandoning without completing, completing without hint?

Also, any new events should be added to the Amplitude file.